stepler/nova/steps/servers.py

```diff
@@ -141,7 +141,9 @@ def attach_floating_ip(self, server, floating_ip, check=True):
 
         if check:
             floating_ips = self.get_ips(server, 'floating').keys()
-            floating_ip.ip in floating_ips
+            assert floating_ip.ip in floating_ips, (
```

Comment:
@AlexanderKoryagin commonly yep, `assert` is simpler (but just wanted to say that `assert_that` can provide custom message too). So let to use `assert` with custom messages until we find something better.